cmake/config.cmake

```diff
@@ -65,7 +65,7 @@ set(USE_AOCL OFF)
 # Whether enable OpenCL runtime
 #
 # Possible values:
-# - ON: enable OpenCL with cmake's auto search
+# - ON: enable OpenCL with OpenCL wrapper
```

Comment:
let's add more wording like 
"... OpenCL wrapper to remove dependency during build time and trigger dynamic search and loading of OpenCL in runtime"

Reply:
Done